lib/src/lib/components/form-fields/select-field/select-field.component.html

```diff
@@ -101,12 +100,3 @@
     }
   </mat-form-field>
 }
-
-@if (fieldIsReadonly && !fieldIsHidden) {
-  <div class="lab900-readonly-field">
-    @if (readonlyLabel(); as label) {
-      <span class="lab900-readonly-field__label">{{ label | translate }}</span>
-    }
-    <div [innerHTML]="readOnlyDisplay()"></div>
-  </div>
-}
```

Comment:
I think removing this will have a significant impact on stip

Reply:
This is already taken care of within the directive